src/main/java/org/broadinstitute/hellbender/tools/sv/cluster/CanonicalSVCollapser.java

```diff
@@ -446,22 +669,16 @@ protected Object collapseSingleVariantAttribute(final Set<Object> values) {
     /***
      * Calculates new SVLEN value.
      * @param items  records to collapse
-     * @param newStart  collapsed start position
-     * @param newEnd   collapsed end position
      * @param newType   collapsed sv type
      * @return
      */
-    protected final int collapseLength(final Collection<SVCallRecord> items, final int newStart, final int newEnd,
-                                       final StructuralVariantType newType) {
+    protected final Integer collapseLength(final Collection<SVCallRecord> items, final StructuralVariantType newType) {
         Utils.nonNull(items);
         Utils.nonEmpty(items);
-        final SVCallRecord exampleCall = items.iterator().next();
-        if (!exampleCall.isIntrachromosomal()) {
-            return -1;
-        } else if (newType.equals(StructuralVariantType.INS)) {
+        if (newType.equals(StructuralVariantType.INS)) {
             return collapseInsertionLength(items);
         } else {
-            return newEnd - newStart + 1;
+            return null;
```

Comment:
So far I've been putting off CPX types since they won't be needed in clustering for gatk-sv. I've added a check to the class that throws an error if it's passed a record with an unsupported SV type.